### DISCUSSION TITLE
Quantifiable bad habits (or: permission to break good habit)

### DISCUSSION THREAD

**Discussion by MarcinGil** [iSoron/uhabits#2061] [Help & FAQ]
Hi!
The recommendation in FAQ is to rephrase bad habit into a good habit, like "don't eat junk food" to "did you have healthy food".
However I'm not that strict, and for example I'd like to have a habit streak but allow for junk food 3 times a month. Or sweets once a week - and still count that against positive habit of min sweets and min junk food.

For now I've set "zero sweets" as 6 / week and "zero junk food" as 25 / month. But thats different to "Junk food allowed 3 / month", and marking days where you didn't have them, be on track if you didn't mark 3 days in month.

Any recommendations how to nicely track it?

**Comment by KyleSCraig** [comment]
I found one negative habit that I could reframe as positive by putting something I could do instead. And in your case that would be eat well or something. I do like it better positive but I also found that for other things I can't come up with a good positive habit. Not playing games could be filled with whatever else I do but because it's not specific (like eating) then the positives apply to all bad uses of time and it feels too hard.

For my No games habit I have 24 in 27 and I think that works well. I use the hide completed tasks setting, so when I have been doing it well and am therefore "entitled" to playing games, I actually don't see the habit so it's not like it's a reminder to play games. Then if I haven't played games, on subsequent days when the habit does come up I'll backfill based on memory. 

That one I often tick at the start of the day to say I have no intention of doing it and I usually won't because it takes intention to do. Another habit I have is to not touch my face unnecessarily, which can just happen at any moment so I can't use the same strategy and it's relatively demanding to have it in mind so much and be unable to complete it. I'm currently experimenting with a timeframe for that so maybe I tick it off at 1pm and not track later than that - and adjusting the timeframe based on my score.
